index.html

```diff
@@ -36,17 +36,17 @@ <h1 class="color1">Hey there. I'm Francisco<br><span class="color2">I'm a softwa
     <section>
         <!-- Work section -->
         <h2 class="color1">My Recent Works</h2>
-        <div class="div1">
+        <div class="flex-column">
             <hr class="hrstyling">
         </div>
         <br>
         <br>
         <br>
```

Comment:
- [ OPTIONAL] I think it would be better to use CSS properties like margin and padding instead of using so many `<br>` tag for code to be readable and professional. For further reference[ check this](https://stackoverflow.com/questions/12101453/is-there-a-general-rule-of-thumb-for-multiple-br-tags)

Example 
```
mt-1 {
  margin-top : 10px;
}
```

Reply:
Thanks for the advice, I will change it the next time I work on the project. 😃